packages/app/utilities.ts

```diff
@@ -623,6 +629,18 @@ export const getCreatorUsernameFromNFT = (nft?: NFT) => {
     : formatAddressShort(nft.creator_address);
 };
 
+export const getUserDisplayNameFromProfile = (user?: UserType) => {
+  if (!user) return "";
+
+  return user.data.profile.username
+    ? `@${user.data.profile.username}`
+    : user.data.profile.name
```

Comment:
rest is `user.data.profile.name || user.data.profile.wallet_addresses_v2?.[0]?.ens_domain || formatAddressShort(user.data.profile.wallet_addresses_v2?.[0]?.address)` 🤓

Reply:
missed this comment on phone, makes sense!